Vulcan: Sidebar: Design tweaks for new image aspect ratios

ianrohde opened the pull request:
## Issue

Mattia updated the design for the Problem sidebar: https://www.figma.com/file/QVLUEpDVoyiMM4MXEGq1L9/iFixit---Problems?type=design&node-id=201-65526&mode=design#600241124

Let's update to match.

## CR/QA

<details>
Figma:
<img width="1681" alt="Screenshot 2023-11-08 at 1 00 11 PM" src="https://github.com/iFixit/react-commerce/assets/1634505/aefff681-0ba2-4d4a-9f0f-4938f5a23f1b">

Current branch:
<img width="1681" alt="Screenshot 2023-11-08 at 12 57 52 PM" src="https://github.com/iFixit/react-commerce/assets/1634505/78cabcae-8f40-4239-9533-f543e33bf728">

</details>

`Troubleshooting/Garbage_Disposal/Hums/491105#Section_Jammed_Motor`

deltuh-vee commented:
dev_block ✊ 
I think we want the copy to be "Common problems"
![CleanShot 2023-11-08 at 14 38 08](https://github.com/iFixit/react-commerce/assets/5289715/d5d9f9f3-1985-41d7-90e2-589821090fc0)

ianrohde commented:
> I think we want the copy to be "Common problems"

Fixed in https://github.com/iFixit/react-commerce/pull/2090/commits/e2f0d04340fd5d485b5b6cc2d13eb36f0783c5df

> and have the section shifted over to be under the dropdown

@deltuh-vee if we zoom out a bit and look at the various design widths, we don't always cap the right side of the content. I added https://github.com/iFixit/react-commerce/pull/2090/commits/1cc2b3ec37f32903905641e498091da3cf674580 which adjusts the main content to the center of the viewport when there is enough space to do so. The calculation is correct, but the alignment doesn't match the deisgn and it aligns to the left of the breadcrumb text instead. This is due to the breadcrumb container padding (second detail):
<details>
<img width="1840" alt="Screenshot 2023-11-09 at 2 04 36 PM" src="https://github.com/iFixit/react-commerce/assets/1634505/3d09a668-8803-4864-94e9-753b6ecc2071">

<img width="1681" alt="Screenshot 2023-11-09 at 2 10 48 PM" src="https://github.com/iFixit/react-commerce/assets/1634505/32bc156a-ce87-46f2-990b-49dd722cf812">

</details>

deltuh-vee commented:
QA 🎬 
Related problems sidebar now matches the mockup a little more closely.

ianrohde commented:
@deltuh-vee why did you change capitalization without communication on https://github.com/iFixit/react-commerce/pull/2090/commits/955c8e0bb7a24fed2bb97aac24b5a15c5f6787b1? The Figma comps are often not grammatically correct.

deltuh-vee commented:
@ianrohde 
Sorry about that. I didn't realize we would want to deviate from the mockup.

ianrohde commented:
> Sorry about that. I didn't realize we would want to deviate from the mockup.

All good, thanks for reverting 👍

hcutrone reviewed (approved):
CR 🌯

ianrohde commented:
Correcting the conflicts in https://github.com/iFixit/react-commerce/pull/2090/commits/99d6c6705b3913cfaa71f089c42e96a41b702c8c was a manual process to get right. Apologies if it is confusing at all.

erinemay commented:
Everything looks great and according to the mockup except for a few small details.

dev_block 🌵 The device title isn't vertically centered in its box. 

| [here](https://react-commerce-prod-git-vulcan-sidebar-design-twe-032f62-ifixit.vercel.app/Troubleshooting/Google_Phone/Google+Pixel+Overheating/479437) | [main](https://react-commerce-prod.vercel.app/Troubleshooting/Google_Phone/Google+Pixel+Overheating/479437#Section_See_Also) | mockup |
| - | - | - |
| ![image](https://github.com/iFixit/react-commerce/assets/72166715/7ea84c58-51d7-48be-b4d2-90eb47ba6dae) | ![image](https://github.com/iFixit/react-commerce/assets/72166715/21c05a76-13db-4b0a-922d-7899293ab7cf) | ![image](https://github.com/iFixit/react-commerce/assets/72166715/576ffe87-192a-4689-a7c5-1a26f4e03f0d)

⬇️ This might not be in scope here. It's not changed from main, but does deviate from the mocked up related problem cards. Happy to make a new issue. @ianrohde, let me know. deploy_block 🍰 

This is the intermediate screen-width state where cards are at the bottom of the page, but not mobile. The way the cards are now matches the wide-width cards when they should match the mid and small width cards.
   - Mockup:
![image](https://github.com/iFixit/react-commerce/assets/72166715/dc020246-358b-466b-99be-65466511a00d)
   - https://react-commerce-prod-git-vulcan-sidebar-design-twe-032f62-ifixit.vercel.app/Troubleshooting/Google_Phone/Google+Pixel+Overheating/479437#Section_See_Also:
![image](https://github.com/iFixit/react-commerce/assets/72166715/99abf1ac-da54-4027-94f3-cf425e88cfed)

ianrohde commented:
@erinemay I'm not able to replicate on MacOS Chromium. What browser/OS were those screenshots from?

ianrohde commented:
> The device title isn't vertically centered in its box.

Fixed in https://github.com/iFixit/react-commerce/pull/2090/commits/2e58464874b34dd93601f1784fa2170d6be0d4f7

> This is the intermediate screen-width state where cards are at the bottom of the page, but not mobile. The way the cards are now matches the wide-width cards when they should match the mid and small width cards.

Fixed in https://github.com/iFixit/react-commerce/pull/2090/commits/62ad52e793f16a631826effdb47086a9035cf6fa

un_dev_block 🟢 
un_deploy_block 🟢

hcutrone reviewed (approved):
CR 🌯

erinemay commented:
QA 🦖 [Issues](https://github.com/iFixit/react-commerce/pull/2090#issuecomment-1811577299) are fixed. Looks great, and reflects the mockup in desktop, tablet, and mobile.
